src/app/process-editor/store/process-editor.effects.ts

```diff
@@ -129,6 +132,20 @@ export class ProcessEditorEffects extends BaseEffects {
         mergeMap(processId => this.deleteProcess(processId))
     );
 
+    @Effect()
+    updateProcessExtensionsEffect = this.actions$.pipe(
+        ofType<UpdateProcessExtensionsAction>(UPDATE_PROCESS_EXTENSIONS),
+        withLatestFrom(this.store.select(selectOpenedModel)),
+        mergeMap(([action, process]) => {
```

Comment:
This seems to be quite hacky... processId of this actioj should be mandatory, instead of optional. We know the process id from the component, where we trigger this action. More precisely from the template, where we use process$ | async. We could pass down the process id, so we don't need this effect at all.